packages/ui/src/components/EditableText/EditableText.tsx

```diff
@@ -237,7 +242,7 @@ export const EditableText = (props: EditableTextProps) => {
   };
   const labelText = (
     <H1Header
-      className={classes.root}
+      className={`${classes.root} ${isBreadcrumb ? 'breadcrumb-text' : ''}`}
```

Comment:
We should probably use the `cx` utility to concatenate classnames. I think it essentially does the same thing, but it's a bit neater.  There are a lot of instances in this file where we can make this change. 

```ts
      className={cx(classes.root, { ['breadcrumb-text']: isBreadcrumb })}
```

```suggestion
      className={cx(classes.root, { ['breadcrumb-text']: isBreadcrumb })}
```

Reply:
Added